packages/launchpad/src/signingcosmosclient.spec.ts

```diff
@@ -172,4 +179,109 @@ describe("SigningCosmosClient", () => {
       assertIsBroadcastTxSuccess(result);
     });
   });
+
+  describe("sign", () => {
+    it("works", async () => {
+      pendingWithoutLaunchpad();
+      const wallet = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic);
+      const client = new SigningCosmosClient(launchpad.endpoint, faucet.address0, wallet);
+
+      const msg1: MsgDelegate = {
+        type: "cosmos-sdk/MsgDelegate",
+        value: {
+          delegator_address: faucet.address0,
+          validator_address: launchpad.validator.address,
+          amount: coin(1234, "ustake"),
+        },
+      };
+      const msg2: MsgSend = {
+        type: "cosmos-sdk/MsgSend",
+        value: {
+          from_address: faucet.address0,
+          to_address: makeRandomAddress(),
+          amount: coins(1234567, "ucosm"),
+        },
+      };
+      const fee = {
+        amount: coins(2000, "ucosm"),
+        gas: "180000", // 180k
+      };
+      const memo = "Use your power wisely";
+
+      const signed = await client.sign([msg1, msg2], fee, memo);
+      expect(signed.msg).toEqual([msg1, msg2]);
+      expect(signed.fee).toEqual(fee);
+      expect(signed.memo).toEqual(memo);
+      expect(signed.signatures).toEqual([
+        {
+          pub_key: faucet.pubkey0,
+          signature: jasmine.stringMatching(base64Matcher),
+        },
+      ]);
+
+      const broadcastResult = await client.broadcastTx(signed);
+      assertIsBroadcastTxSuccess(broadcastResult);
+    });
+  });
+
+  describe("appendSignature", () => {
+    it("works", async () => {
+      pendingWithoutLaunchpad();
+      const wallet0 = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic, makeCosmoshubPath(0));
+      const wallet1 = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic, makeCosmoshubPath(1));
+      const client0 = new SigningCosmosClient(launchpad.endpoint, faucet.address0, wallet0);
+      const client1 = new SigningCosmosClient(launchpad.endpoint, faucet.address1, wallet1);
+
+      const msg1: MsgSend = {
+        type: "cosmos-sdk/MsgSend",
+        value: {
+          from_address: faucet.address0,
+          to_address: makeRandomAddress(),
+          amount: coins(1234567, "ucosm"),
+        },
+      };
+      const msg2: MsgSend = {
+        type: "cosmos-sdk/MsgSend",
+        value: {
+          from_address: faucet.address1,
+          to_address: makeRandomAddress(),
+          amount: coins(1234567, "ucosm"),
+        },
+      };
+      const fee = {
+        amount: coins(2000, "ucosm"),
+        gas: "160000", // 2*80k
+      };
+      const memo = "This must be authorized by the two of us";
+
+      const signed = await client0.sign([msg1, msg2], fee, memo);
```

Comment:
So the API is:

```ts
      const signed = await client0.sign([msg1, msg2], fee, memo);
      const cosigned = await client1.appendSignature(signed);
      const broadcastResult = await client0.broadcastTx(cosigned);
```

It would be nice to document that somewhere (the README), along with the typical case:

```ts
const result = await client.signAndBroadcast([msg], fee, "Use your power wisely");
```

Quite simple to use actually